docs/section9/Rule6-jb.md

```diff
@@ -13,24 +13,36 @@
 **Data Lookup:** None  
 ## Rule Logic: 
 
-- Calculate the total lighting power for each space under each building segment in the User RMR: ```For building_segment in U_RMR.building.building_segments:``` (Note XC, right now in the schema, interior lighting is independent of the space it is in. It is only related to space type. Hard to calculate the lighting power for each space.)
+- Calculate the total lighting power for each space under each building segment in the User RMR: ```for building_segment_user in U_RMR.building.building_segments:```  
 
-  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+  - For each thermal_block in building segment: ```thermal_block_user in building_segment_user.thermal_blocks:```
 
-  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
+  - For each zone in thermal block: ```zone_user in thermal_block_user.zones:```
 
-  - Get space from thermal zone: ```space in thermal_zone.spaces:```
+  - For each space in thermal zone: ```space_user in zone_user.spaces:```  
 
-  - Get the total lighting power for each space: ```space_lighting_power_user = space.lighting_power``` (Note XC, depending on how rmr reports the lighting power for each space, e.g. report fixtures, or report total wattage, this needs to be adjusted.)
+    - Get floor_area from space: ```floor_area_user = space_user.floor_area```  
 
-- Calculate the total lighting power for each space under each building segment in the Proposed RMR: ```For building_segment in P_RMR.building.building_segments:```
+    - Get interior_lighting in space: ```interior_lighting_user = space_user.interior_lightings```  
```

Comment:
I believe it should be `space_user.interior_lighting`

Reply:
Thanks Charlie. This is now corrected.

Reply:
Thanks.